.yamato/config/__shared.metafile

```diff
@@ -27,20 +27,101 @@ build_configs:
+      - [Android_OpenGLES3, Android_Vulkan, iPhone_Metal]: --player-save-path=players  
+      - [all]: --build-only
+      - [all]: --timeout=1200
+      - [Android_OpenGLES3, Android_Vulkan]: --timeout=2700
+      - [iPhone]: --timeout=1800
+      - [Linux_OpenGlCore]: --extra-editor-arg="-executemethod CustomBuild.BuildLinuxOpenGlCoreLinear"
+      - [Linux_Vulkan]: --extra-editor-arg="-executemethod CustomBuild.BuildLinuxVulkanLinear"
+      - [Win_DX11]: --extra-editor-arg="-executemethod CustomBuild.BuildWindowsDX11Linear"
+      - [Win_DX12]: --extra-editor-arg="-executemethod CustomBuild.BuildWindowsDX12Linear"
+      - [Win_Vulkan]: --extra-editor-arg="-executemethod CustomBuild.BuildWindowsVulkanLinear"
+      - [all]: --scripting-backend=<SCRIPTING_BACKEND> 
+      - [all]: --extra-editor-arg="-colorspace=<COLORSPACE>"
+      - [Win_DX11]: --extra-editor-arg="-playergraphicsapi=Direct3D11"
+      - [Win_DX12]: --extra-editor-arg="-playergraphicsapi=Direct3D12"
+      - [Win_Vulkan, Linux_Vulkan, Android_Vulkan]: --extra-editor-arg="-playergraphicsapi=Vulkan"
+      - [Android_OpenGLES3]: --extra-editor-arg="-playergraphicsapi=OpenGLES3"
+      - [iPhone_Metal]: --extra-editor-arg="-playergraphicsapi=Metal"
+  - type: playmode
+    utr_flags:
+      - [all]: --suite=playmode
+      - [all]: --testproject=.
+      - [OSX_Metal]: --testproject=/Users/bokken/Graphics/TestProjects/<PROJECT_FOLDER>
+      - [all]: --editor-location=.Editor
+      - [OSX_Metal]: --editor-location=/Users/bokken/.Editor
+      - [all]: --artifacts_path=test-results
+      - [OSX_Metal]: --artifacts_path=/Users/bokken/Graphics/TestProjects/<PROJECT_FOLDER>/test-results
+      - [all]: --reruncount=2
+      - [Win_DX11]: --extra-editor-arg="-force-d3d11"
+      - [Win_DX12]: --extra-editor-arg="-force-d3d12"
+      - [Win_Vulkan, Linux_Vulkan]: --extra-editor-arg="-force-vulkan"
+      - [Linux_OpenGlCore]: --extra-editor-arg="-force-glcore"
+      - [all]: --scripting-backend=<SCRIPTING_BACKEND> 
+      - [all]: --extra-editor-arg="-colorspace=<COLORSPACE>"
+  - type: editmode
+    utr_flags:
+      - [all]: --suite=editor
+      - [all]: --platform=editmode
+      - [iPhone_Metal]: --platform=iOS
+      - [all]: --testproject=.
+      - [OSX_Metal]: --testproject=/Users/bokken/Graphics/TestProjects/<PROJECT_FOLDER>
+      - [all]: --editor-location=.Editor
+      - [OSX_Metal]: --editor-location=/Users/bokken/.Editor
+      - [all]: --artifacts_path=test-results
+      - [OSX_Metal]: --artifacts_path=/Users/bokken/Graphics/TestProjects/<PROJECT_FOLDER>/test-results
+      - [all]: --reruncount=2
+      - [Win_DX11]: --extra-editor-arg="-force-d3d11"
+      - [Win_DX12]: --extra-editor-arg="-force-d3d12"
+      - [Win_Vulkan, Linux_Vulkan]: --extra-editor-arg="-force-vulkan"
+      - [Linux_OpenGlCore]: --extra-editor-arg="-force-glcore"
+      - [all]: --scripting-backend=<SCRIPTING_BACKEND> 
+      - [all]: --extra-editor-arg="-colorspace=<COLORSPACE>"
```

Comment:
maybe this stuff can have some code comments and line breaks in it to help clean it up and organise it? it is a bit overwhelming to see a big list like this :D

Reply:
agreed, it is massive :D I'll see what I can do!

Reply:
I've grouped them together now (first flags applying for all, then groups per flag). Not sure if the spaces inbetween add noise or reduce it though

Reply:
i like the space in between! maybe we also want comments to serve as headers eg #### platform, #### player_load_path, etc

Reply:
i was thinking too, but was afraid it gets too cramped, especially with flags only defined once.. we can always add it later though too!